### PR TITLE
Feature: Databricks compatibility

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
Confirm the following files were correctly updated automatically:
- [x] `hooks/pre-command` line added
- [x] `pipeline.yml` section added
- [x] `sample.profiles.yml` catalog updated
- [x] `integration_tests/dbt_project.yml` section added (check for dupes though)
- [x] Version updates for `dbt_project.yml` and `integration_tests/dbt_project.yml` (automatically moved to next breaking change)
- [x] Changelog (automatically moved to next breaking change, but check the PR links were added correctly!) 

Manual updates are:
- [x] README
- [x] `packages.yml` for transform packages
- [ ] ~Incremental Models~
- [x] Regen Docs
- [x] Local Databricks run and test passing
- [x] Buildkite passing

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 packages:
-  - package: fivetran/quickbooks_source
-    version: [">=0.8.0", "<0.9.0"]
+  # - package: fivetran/quickbooks_source
+  #   version: [">=0.9.0", "<0.10.0"]
+
+- git: https://github.com/fivetran/dbt_quickbooks_source.git
+  revision: MagicBot/databricks-compatibility
+  warn-unpinned: false
```
Comment (fivetran-catfritz) [inline packages.yml:7]: Update before merge/release.

File: integration_tests/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 dbt-snowflake>=1.3.0,<2.0.0
 dbt-bigquery>=1.3.0,<2.0.0
-dbt-redshift>=1.3.0,<2.0.0
+dbt-redshift>=1.3.0,<1.5.0
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/requirements.txt:3]: Can we bump this back up

**Review by fivetran-joemarkiewicz — APPROVED** [review]
lgtm

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,7 @@
 packages:
-  - package: fivetran/quickbooks_source
-    version: [">=0.8.0", "<0.9.0"]
\ No newline at end of file
+  # - package: fivetran/quickbooks_source
+  #   version: [">=0.9.0", "<0.10.0"]
+
+- git: https://github.com/fivetran/dbt_quickbooks_source.git
+  revision: MagicBot/databricks-compatibility
+  warn-unpinned: false
```
Comment (fivetran-catfritz) [inline packages.yml:7]: Update before release.

File: integration_tests/requirements.txt (hunk 2 above)
Comment (fivetran-catfritz) [reply integration_tests/requirements.txt]: bumped!

**Pull request merged**